incubator/istio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Istio Helm chart for Kubernetes
 name: istio
-version: 0.2.1
+version: 0.2.2
```

Comment:
Given the added scope of changes lets make this 0.3.0 (apologies for the scope creep)